src/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+add_executable(app 
+    main.cpp
+)
+
+add_subdirectory(engine)
+add_subdirectory(game)
+add_subdirectory(settings)
+
+target_include_directories(app PRIVATE ${PROJECT_SOURCE_DIR}/include)
+
+find_package(SFML 2 REQUIRED network audio graphics window system) 
```

Comment:
why `network` and `audio` is it for the future improvements?

Reply:
Probably be used for future improvements, but useless for now you are right
Removed in a0a25cd99afb9de77793b8f13f0ca9cd1f8abc2a